lib/bakeAmbientOcclusion.js

```diff
@@ -584,3 +1074,43 @@ function sumBarycentric(barycentric, vector0, vector1, vector2, result) {
     Cartesian3.add(result, sumBarycentricScratch, result);
     return result;
 }
+
+var xAxis = new Cartesian3();
+var zAxis = new Cartesian3();
+var yAxis = new Cartesian3();
+// Returns the point's x/y coordinates in the triangle's plane where position0 is the origin
+// and position1 - position0 is the "x" axis.
+// http://paulbourke.net/geometry/pointlineplane/
+function flattenTriangle(positions, results) {
```

Comment:
The stuff from the Paul Bourke site is all just math, should I keep the reference here if there's no code?